workflows/pipe-common/pipeline/api/api.py

```diff
@@ -1140,6 +1147,76 @@ def prolong_lifecycle_rule(self, datastorage_id, rule_id, path, days, force=Fals
             raise RuntimeError("Failed to prolong lifecycle rule '{}'.",
                                "Error message: {}".format(str(rule_id), str(e.message)))
 
+    def load_datastorage_item_content(self, storage_id, path, version=None):
+        try:
+            formed_url = self.LOAD_STORAGE_ITEM_CONTENT_URL.format(id=storage_id, path=path)
+            if version:
+                formed_url = formed_url + "&version={version}".format(version=version)
+            return self.execute_request(str(self.api_url) + formed_url)
```

Comment:
It is better to use `self._request` method rather than `self.execute_request` because it provides more efficient error handling. The same applies for all the methods below.